plugins/hive/tests/test_hive_task.py

```diff
@@ -1,13 +1,13 @@
 import pandas
 import pytest
+from plugins.hive.flytekitplugins.hive.task import HiveConfig, HiveSelectTask, HiveTask
```

Comment:
I can understand this absolute import but still a bit strange.

Reply:
oh no this is wrong, i'll fix.